lib/pbench/cli/server/user_create.py

```diff
@@ -29,9 +30,14 @@ def execute(self):
             first_name=self.context.first_name,
             last_name=self.context.last_name,
             email=self.context.email,
+            role=self.context.role if self.context.role else "",
```

Comment:
I'm happy to take your word for it, but just to confirm:  an empty string is the appropriate value for `role` if it is unspecified?  We want it to be represented in the database as "null" -- is that how we attain that?  (I would have thought that the appropriate value would be 'None`, but I'm not that familiar with SQLAlchemy....)

Reply:
If you look at line 80 of `lib/pbench/server/database/models/users.py` file (where we `validate` role column value) the return statement is like this `return Roles[value.upper()]` where we convert the role value to upper case and catch the `keyError` if no such a key is present in our Roles enum class. However, if the value passed is `None` it raises the attribute exception like NoneType object has no attribute upper(), that wont be the case if we pass an empty string as a role value (It will be saved in our table as None anyway because no empty key is present in our enum class). Now indeed it can also be fixed by catching a generic `Exception` but I thought if we can catch a specific error instead of generic exception that would be better.

Reply:
@webbnh are you good with this answer?

Reply:
I forgot about the field validator.

So, if the role string is not a valid role enum, then the field validator is going to return `None`, and the database entry will quietly be set to "null", which is what we want to have happen here.  We just need to be cognizant of this silent failure mode and be aware that we're relying on the command line processing to catch and report it.

OK, merge it!